applications/nrf_desktop/README.rst

```diff
@@ -1556,6 +1556,8 @@ The B0 bootloader requires the following options enabled:
 * :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` - Required for providing the signature used for image signing and verification.
 * :kconfig:option:`CONFIG_FW_INFO` - Required for the application versioning information.
 * :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` - Enable this option to set the version of the application after you enabled :kconfig:option:`CONFIG_FW_INFO`.
+  For the DFU package generation, you need to update this configuration in the main application image and the ``s1_image`` child image.
```

Comment:
Maybe we should explain here that we build images for both S0 and S1 if we use B0. The S0 image is built as the main application and S1 image is built as a child image called `s1_image`. Such introduction could be helpful for customers.